webpack.dev.config.js

```diff
@@ -51,13 +51,19 @@ module.exports = {
     mainFields: ['browser', 'jsnext:main', 'main']
   },
   plugins: [
-    new webpack.EnvironmentPlugin(['DUCOR_API_URL', 'DUCOR_API_PORT']),
+    new EnvironmentPlugin(['DUCOR_API_URL', 'DUCOR_API_PORT']),
     new CleanWebpackPlugin([ 'dist' ]),
     new HtmlWebpackPlugin({
       title: 'Hot Module Replacement',
       template: 'src/index.html'
     }),
-    new webpack.HotModuleReplacementPlugin(),
+    new HotModuleReplacementPlugin(),
+    new DefinePlugin({
+      'process.env': {
+        NODE_ENV: JSON.stringify('development'),
```

Comment:
Why use `JSON.stringify()` on a string?

Reply:
Because you should wrap string into string, for example: " 'development' ", and only this way is working.
We use JSON.stringify for all types (objects, arrays, strings, numbers) instead of unobvious "'"' for strings only.

Reply:
Looks like an anti-pattern, but let's leave it as it is for now.